File: src/main/java/com/example/stockfx/entities/Magazine.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.example.stockfx.entities;

import javafx.beans.property.SimpleStringProperty;

/**
 *
 * @author nours
 */
public class Magazine {
    private String titre_magazine,genre_magazine,image_magazine,description_magazine;

    public void setTitre_magazine(String titre_magazine) {
        this.titre_magazine = titre_magazine;
    }

    public void setGenre_magazine(String genre_magazine) {
        this.genre_magazine = genre_magazine;
    }

    public void setImage_magazine(String image_magazine) {
        this.image_magazine = image_magazine;
    }

    public void setDescription_magazine(String description_magazine) {
        this.description_magazine = description_magazine;
    }
    int ref;

    public void setRef(int ref) {
        this.ref = ref;
    }

    public int getRef() {
        return ref;
    }
    

    public String getTitre_magazine1() {
        return titre_magazine;
    }

    public String getGenre_magazine1() {
        return genre_magazine;
    }

    public String getImage_magazine1() {
        return image_magazine;
    }

    public String getDescription_magazine1() {
        return description_magazine;
    }

    public Magazine(String titre_magazine, String genre_magazine, String image_magazine, String description_magazine) {
        this.titre_magazine = titre_magazine;
        this.genre_magazine = genre_magazine;
        this.image_magazine = image_magazine;
        this.description_magazine = description_magazine;
    }

    

    public Magazine() {
           }

    public SimpleStringProperty getTitre_magazine() {
        return new SimpleStringProperty(titre_magazine);
    }
 
   /* public void setTitre_magazine(String titre_magazine) {
       this.titre_magazine = new SimpleStringProperty(titre_magazine);
    }*/

    public SimpleStringProperty getGenre_magazine() {
        return new SimpleStringProperty(genre_magazine);
    }

   /* public void setGenre_magazine(String genre_magazine) {
         this.genre_magazine = new SimpleStringProperty(genre_magazine);
    }*/

    public SimpleStringProperty getImage_magazine() {
        return new SimpleStringProperty(image_magazine);
    }

   /* public void setImage_magazine(String image_magazine) {
         this.image_magazine = new SimpleStringProperty(image_magazine);
    }*/

    public SimpleStringProperty getDescription_magazine() {
        return new SimpleStringProperty(description_magazine);
    }

   /* public void setDescription_magazine(String description_magazine) {
         this.description_magazine = new SimpleStringProperty(description_magazine);
    }*/

    @Override
    public String toString() {
        return "ref: "+ref+" ,titre_magazine=" + titre_magazine + ", genre_magazine=" + genre_magazine + ", image_magazine=" + image_magazine + ", description_magazine=" + description_magazine + '}';
    }
    
    
}

File: src/Entity/Magazine.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Magazine
 *
 * @ORM\Table(name="magazine")
 * @ORM\Entity
 */
class Magazine
{
    /**
     * @var int
     *
     * @ORM\Column(name="ref_magazine", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $refMagazine;

    /**
     * @var string
     *
     * @ORM\Column(name="titre_magazine", type="string", length=50, nullable=false)
     */
    private $titreMagazine;

    /**
     * @var string
     *
     * @ORM\Column(name="genre_magazine", type="string", length=50, nullable=false)
     */
    private $genreMagazine;

    /**
     * @var string
     *
     * @ORM\Column(name="image_magazine", type="string", length=255, nullable=false)
     */
    private $imageMagazine;

    /**
     * @var string
     *
     * @ORM\Column(name="description_magazine", type="string", length=100, nullable=false)
     */
    private $descriptionMagazine;

    /**
     * @var int
     *
     * @ORM\Column(name="selected", type="integer", nullable=false)
     */
    private $selected = '0';

    public function getRefMagazine(): ?int
    {
        return $this->refMagazine;
    }

    public function getTitreMagazine(): ?string
    {
        return $this->titreMagazine;
    }

    public function setTitreMagazine(string $titreMagazine): self
    {
        $this->titreMagazine = $titreMagazine;

        return $this;
    }

    public function getGenreMagazine(): ?string
    {
        return $this->genreMagazine;
    }

    public function setGenreMagazine(string $genreMagazine): self
    {
        $this->genreMagazine = $genreMagazine;

        return $this;
    }

    public function getImageMagazine(): ?string
    {
        return $this->imageMagazine;
    }

    public function setImageMagazine(string $imageMagazine): self
    {
        $this->imageMagazine = $imageMagazine;

        return $this;
    }

    public function getDescriptionMagazine(): ?string
    {
        return $this->descriptionMagazine;
    }

    public function setDescriptionMagazine(string $descriptionMagazine): self
    {
        $this->descriptionMagazine = $descriptionMagazine;

        return $this;
    }

    public function getSelected(): ?int
    {
        return $this->selected;
    }

    public function setSelected(int $selected): self
    {
        $this->selected = $selected;

        return $this;
    }


}
